### PR TITLE
Bring installation instructions up to date

### PR DESCRIPTION
**Pull request by rtdietrich**
hi I'm a rookie and i suggest a few changes in your install instructions because the actual version didnt work for me and a had to google a lot to make it run. All i wrote below is tested on a fresh Ubuntu 16.04.1
so here are my suggestions (be also aware of "cd ~/ponyc/" at the end of text:

### Ubuntu (12.04, 14.04, 15.10, 16.04)

Install and setup Git (if not already done):

Get an Account here: https://github.com/
```bash
$ sudo apt install git
$ git config --global user.email "you@example.com"
$ git config --global user.name "Your Name"
 ```

Get Pony-Sources:

```bash
$ git clone git://github.com/ponylang/ponyc
 ```

Ubuntu and some other Linux distributions don't include pcre2 in their package manager. pcre2 is used by the Pony regex package. To download and build pcre2 from source:

 ```bash
 $ wget ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre2-10.21.tar.bz2
 $ tar xvf pcre2-10.21.tar.bz2
 $ cd pcre2-10.21
 $ ./configure --prefix=/usr
 $ make
 $ sudo make install
 ```
 
 To build ponyc, compile and run helloworld:
 
 ```bash
 $ cd ~/ponyc/
 $ make config=release
 $ ./build/release/ponyc examples/helloworld
 $ ./helloworld
 ```

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -48,7 +50,9 @@ layman -a stefantalpalaru
 emerge dev-lang/pony
 ```
 
-A live ebuild is also available in the [overlay](https://github.com/stefantalpalaru/gentoo-overlay) (dev-lang/pony-9999) and for Vim users there's app-vim/pony-syntax.
+A live ebuild is also available in the
+[overlay](https://github.com/stefantalpalaru/gentoo-overlay)
+(dev-lang/pony-9999) and for Vim users there's app-vim/pony-syntax.
```
Comment (SeanTAllen) [inline getting-started/installation.md]: can you undo the hard wrap you added here?

File: getting-started/installation.md [hunk 2]
```diff
@@ -32,7 +32,9 @@ docker run -v /path/to/my-code:/src/main ponylang/ponyc ./main
 
 ## Mac OS X using [Homebrew](http://brew.sh)
 
-The homebrew version is currently woefully out of date. We are transitioning to a new release system that will keep homebrew up to date. For now, please build from source.
+The homebrew version is currently woefully out of date. We are transitioning to
+a new release system that will keep homebrew up to date. For now, please build
+from source.
```
Comment (SeanTAllen) [inline getting-started/installation.md]: can you undo the hard wrap you added here?

Comment (SeanTAllen) [inline getting-started/installation.md]: This is also dated and now incorrect.

**Comment by SeanTAllen** [comment]
There are a lot of formatting changes here that aren't needed. Its really hard to see what the actual changes are.

**Comment by SeanTAllen** [comment]
This content was removed from the tutorial and now points to the README in ponyc. You appear yo be using a dated version of the tutorial?

**Comment by rtdietrich** [comment]
i maybe found one more "bug" in the install instuctions. to work i had to insert "sudo apt-get update" here:

Linux using a DEB package (via Bintray)

For Ubuntu or Debian Linux, the master and release branches are packaged and availabe on Bintray (pony-language/ponyc-debian).

To install release builds via Apt:

echo "deb https://dl.bintray.com/pony-language/ponyc-debian pony-language main" | sudo tee -a /etc/apt/sources.list

**sudo apt-get update**

sudo apt-get install ponyc-release
Or, for master builds:

sudo apt-get install ponyc-master

**Comment by SeanTAllen** [comment]
the instructions you are updating dont exist anymore. the latest are in https://github.com/ponylang/ponyc

Can you compare to the latest there and if anything is problematic PR against that?

**Comment by rtdietrich** [comment]
i will try to do it, but i'm really very new to such things like guthub

2016-09-20 2:49 GMT+02:00 Sean T Allen <notifications@github.com>:

**Comment by rtdietrich** [comment]
can you help me please with a more exactly link where to find it?

2016-09-20 2:51 GMT+02:00 Ralf Dietrich <rtdietrich@googlemail.com>:

**Comment by SeanTAllen** [comment]
All the installation instructions are in README.md in the ponyc repo:

https://github.com/ponylang/ponyc/blob/master/README.md

To issue a PR, you will want to create a fork of that repo.
Create your own branch.
Make any updates there.
Issue a PR from your branch to ponyc/master

**Comment by rtdietrich** [comment]
ty for your help.... i do my best

2016-09-20 2:58 GMT+02:00 Sean T Allen <notifications@github.com>:

**Comment by rtdietrich** [comment]
ok... my work is done (i hope) Can you see it now!?
i fork... branch...update the "readme" and finally create (i hope the
right) PR

2016-09-20 2:59 GMT+02:00 Ralf Dietrich <rtdietrich@googlemail.com>:

**Pull request closed without merging**